gravitee-am-model/src/main/java/io/gravitee/am/model/User.java

```diff
@@ -161,6 +160,8 @@ public class User implements IUser {
 
     private Boolean forceResetPassword = Boolean.FALSE;
 
+    private Boolean serviceAccount;
```

Comment:
```suggestion
    private Boolean serviceAccount = Boolean.FALSE;
```

shouldn't it be FALSE as default?